openpype/hosts/houdini/plugins/publish/extract_composite.py

```diff
@@ -45,8 +46,11 @@ def process(self, instance):
             "frameEnd": instance.data["frameEndHandle"],
         }
 
-        from pprint import pformat
-
-        self.log.info(pformat(representation))
+        # inject colorspace data
```

Comment:
```suggestion
        # Inject colorspace with 'scene_linear' as that's the default Houdini
        # working colorspace and all extracted images should be in that
        # colorspace
```

Reply:
however, is this actually true, isn't it possible to extract any other colorspace? I'd imagine this would only be the case if it's .exr but not other outputs?

Reply:
I agree! 

https://www.sidefx.com/docs/houdini/render/linear.html#image-formats

Reply:
any way's I'll inject that color space only if it's exr, as I don't know function in python api that's capable of that.